File: 프로그래머스/unrated/181910. 문자열의 뒤의 n글자/문자열의 뒤의 n글자.java
class Solution {
    public String solution(String my_string, int n) {
        String answer = "";
        int tmp = my_string.length() - n;
        answer = my_string.substring(tmp);
        // my_string의 전체 길이에서 n을 뺀값 만큼 출력해야
        // 전체 문자열의 뒷 n글자이므로 새로운 변수로 선언하여
        // 출력한다
        return answer;
    }
}

File: 프로그래머스/0/181910. 문자열의 뒤의 n글자/문자열의 뒤의 n글자.js
function solution(my_string, n) {
    var answer = "";
    let tmp = my_string.length - n;
    answer = my_string.substring(tmp);
    // my_string의 전체 길이에서 n을 뺀 값만큼 출력해야
    // 전체 문자열의 뒷 n글자이므로 새로운 변수로 선언하여
    // 출력한다
    return answer;
}
